projects/plugins/jetpack/extensions/blocks/contact-form/index.js

```diff
@@ -16,6 +16,7 @@ import deprecated from './deprecated';
 import edit from './edit';
 import transforms from './transforms';
 import variations from './variations';
+import './view';
```

Comment:
If you'd like to start loading js on the frontend whenever the form block is added to a post / page, you would need to also rely on `Jetpack_Gutenberg::load_assets_as_required()` in PHP to ensure the js gets loaded on the frontend.

That said, I'm thinking we may not need to add this to the frontend, right? It should be enough to load this in the editor.
In this case, I would recommend that you move the code from `view.js` to `edit.js`.
This will have the added bonus of not having to load `@wordpress/data` on the frontend.

Reply:
> That said, I'm thinking we may not need to add this to the frontend, right?

Yes this is not needed in the front-end only the editor. It's purpose is to update existing form blocks to remove the delete option for the submit button.

> In this case, I would recommend that you move the code from view.js to edit.js.
> This will have the added bonus of not having to load @wordpress/data on the frontend.

Thank you for the suggestion, this make sense since this piece of code is not needed in the front-end.